docs/docsource/Reference/bucket_operations/put_bucket_replication.rst

```diff
@@ -58,10 +58,10 @@ status, and details about the destination.
 The destination details include the bucket in which to store replicas
 and optional storage classes to use to store the replicas.
 
-S3 acts only on rules with the status Enabled. The configuration also
-identifies an IAM role for S3 to assume for copying objects. This role
+Zenko only acts on rules with an Enabled status. The configuration also
+identifies an IAM role for Zenko to assume for copying objects. This role
```

Comment:
Not related to this PR but may be fixed in one shot:

The use of IAM roles is S3C-specific, Zenko does not support IAM roles. Instead, we pre-create service accounts, one for each service (OTOH: Replication, Lifecycle, Ingestion, Garbage Collector, Metadata Search). Each service uses keys generated for their respective account to execute the operations.

Reply:
done